### PR TITLE
[JARS-86] added ontology to resource type

### PR DESCRIPTION
**Pull request by prnvshrn**
For this issue, I have added ontology to resource type dropdown. 
**Screenshot 1 : When editing a resource**
screen shot 2018-07-20 at 1 55 19 pm

<br>**Screenshot 2 : When creating a new resource**
screen shot 2018-07-20 at 1 24 31 pm

### CODE REVIEW

File: cookies/forms.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class TypeModelChoiceField(forms.ModelChoiceField):
 
     def label_from_instance(self, obj):
         if obj.schema is not None:
-            return u'%s: %s' % (obj.schema.name, obj.name)
+            return u'%s (%s)' % (obj.name, obj.schema)
```
Comment (jdamerow) [inline cookies/forms.py]: is there a reason you use obj.schema instead of obj.schema.name?

Comment (jdamerow) [reply cookies/forms.py]: @prnvshrn

Comment (prnvshrn) [reply cookies/forms.py]: I was returning the value incorrectly. Changed obj.schema to obj.schema.name

**Comment by jdamerow** [comment]
I get an attribute error when I try to upload a file now:
```'Collection' object has no attribute 'schema'```

**Comment by prnvshrn** [comment]
Changed the label of resource_type in forms from **CustomModelChoiceField** to **TypeModelChoiceField**

**Comment by jdamerow** [comment]
this does not work for adding new metadata
image

**Pull request merged**